website/pages/docs/extending/plugins.mdx

```diff
@@ -81,6 +84,11 @@ The valid types for plugins are:
 - `provisioner` - A provisioner to install software on images created by a
   builder.
 
+~> **Note**: Only _multi-plugin binaries_ are expected to work with Packer's plugin
+manager. The legacy `builder`, `post-processor` and `provisioner` plugin types
+will keep on being detected but Packer cannot install them automatically.
```

Comment:
this is definitely a warning I should add to the packer-plugin-check

Reply:
Totally ! adding that.